responsibleai_text/responsibleai_text/managers/error_analysis_manager.py

```diff
@@ -67,15 +70,23 @@ def __init__(self, model, dataset, is_multilabel, task_type, classes=None):
         self.classes = classes
         self.is_multilabel = is_multilabel
         self.task_type = task_type
-        classif_tasks = [ModelTask.TEXT_CLASSIFICATION,
-                         ModelTask.MULTILABEL_TEXT_CLASSIFICATION]
+        classif_tasks = [
+            ModelTask.TEXT_CLASSIFICATION,
+            ModelTask.MULTILABEL_TEXT_CLASSIFICATION,
+        ]
         if self.task_type in classif_tasks:
             dataset = self.dataset.iloc[:, 0].tolist()
-            self.predictions = self.model.predict(dataset)
-            self.predict_proba = self.model.predict_proba(dataset)
+            self.predictions = self._raise_user_error_on_failure(
```

Comment:
everything else is from black formatter, let me know and I can revert the changes on formatting

Reply:
Shall we move the predict on the entire dataset in responsibleai-text here:- https://github.com/microsoft/responsible-ai-toolbox/blob/798e9d5eb12c35d72aedea0f604753543c26f702/responsibleai_text/responsibleai_text/rai_text_insights/rai_text_insights.py#L245? Moving predict validation in reponsibleai-text makes the validation same across all RAI verticals.

Reply:
discussed on teams. full test data prediction is a very expensive call especially on text data. We can mitigate the issue with good user error exception message for now and think of caching later.